ieeeuoft/newhacks-site-2022
# added newhacks logo to HSS and fixed null hardware constraint displaying

## Overview

- Resolves #NH-24 and #NH-34
- added newhacks logo to navbar in HSS
![image](https://user-images.githubusercontent.com/20618460/199136222-fb777083-888b-4ca2-8837-39fa0d5eff06.png)
- removed "max null of this item per team", if the max_per_team is null the constraint point won't appear



## Unit Tests Created

- none


## Steps to QA

- test to see if the above features are true. you will need to set a max_per_team value of null (or empty) for the hardware item, you can do this thru import or thru the dbshell (lmk if you do it this way,  I can show you how)
